app/models/datafile.rb

```diff
@@ -0,0 +1,13 @@
+class Datafile < ApplicationRecord
+  validates :origFilename,
```

Comment:
i'm not sure that all of these should be validated that `presence: true` -  for instance:

* `dataverseId` will only be populated after the file is successfully added to Dataverse.

the other metadata i'm less sure about - it might be helpful to talk through the job processes.

Reply:
It would probably be worth talking through the processes tomorrow if time permits! For now I'll strip most of the "presence:true" and "NOT NULL" requirements I added in the DB via the migrations.